introduction.md

```diff
@@ -1,3 +1,9 @@
 Hello, my name is Rohit and I am a Software Engineer who is passionate about learning new technologies.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?